Assets/Project/Scripts/GamePlayScene/GamePlayDirector.cs

```diff
@@ -209,10 +209,10 @@ public void PauseButtonDown()
         private static void CleanObject()
         {
             // パネルを破壊
```

Comment:
```suggestion
            // ボトルを破壊
```
あとこの4行上の、メソッドについての説明部分も。